docs/api/instruments/minicircuits/usb_2sp2t_dch.rst

```diff
@@ -0,0 +1,7 @@
+####################################################
+Minicircuits 2 Channel DPST USB Solid State RF relay
+####################################################
+
+.. autoclass:: pymeasure.instruments.minicircuits.usb_2sp2t_dch
```

Comment:
When trying to build the docs of this thing, an warning is thrown which fully prevents building the docs of this class (independent of the capitalization of usb_2sp2t_dch):
```
WARNING: autodoc: failed to import class 'minicircuits.USB_2SP2T_DCH' from module 'pymeasure.instruments'; the following exception was raised:
No module named 'clr'
```

This has to do with the package `pythonnet` not being installed/not being a part of the requirements (causing the inability to import `clr`); I however do not know a good way to fix this. Maybe @bilderbuchi has an idea?

Reply:
I don't know what needs pythonnet, but dll-based communication feels out of scope of pymeasure, considering the resources we have.